src/cms/templates/dashboard/dashboard.html

```diff
@@ -1,5 +1,45 @@
 {% extends "_base.html" %}
 {% load i18n %}
+{% load static %}
+{% load parse_struct_time %}
 {% block content %}
-    Dashboard
+    <div class="dashboard">
+        <h1 class="font-bold">Dashboard</h1>
+        <div class="flex items-start">
+            <div class="flex flex-col w-1/2 ">
+                <div class="bg-white p-4 my-3 mr-3 shadow">
+                    <h2 class="font-bold">{% trans 'At a glance' %}</h2>
+                </div>
+                <div class="bg-white p-4 my-3 mr-3 shadow">
+                    <h2 class="font-bold">{% trans 'Author Chat' %}</h2>
+                </div>
+            </div>
+            <div class="flex flex-col bg-white w-1/2 px-4 mt-3 ml-3 shadow">
+                <button type="button" class="collapsible font-bold py-4 text-left relative focus:outline-none">
+                    <span class="font-bold">Tür an Tür - Digitalfabrik News</span>
+                    <i data-feather="chevron-up" id="up-arrow" class="inline absolute right-0"></i>
+                    <i data-feather="chevron-down" id="down-arrow" class="inline hidden absolute right-0"></i>
+                </button>
+                <div class="collapsible-content flex flex-col active">
+                    <div class="flex py-4">
+                        <img src="{% static 'images/favicon.ico' %}"/>
+                        <div class="flex flex-col mx-auto">
+                            <h3 class="text-gray-700 font-bold">Tür an Tür - Digitalfabrik</h2>
+                            <a href="{{ home_page }}" class="font-bold text-blue-500 hover:text-blue-700">integreat-app.de</a>
```

Comment:
Maybe also replace the shown part of the link? In case the setting changes, this should probably change as well...
If you don't want to show the protocol, you can extract the domain with
```
from urllib.parse import urlparse

domain = urlparse(RSS_FEED_URLS["home-page"]).netloc
```
in the view.

Reply:
> Maybe also replace the shown part of the link? In case the setting changes, this should probably change as well...
> If you don't want to show the protocol, you can extract the domain with
> 
> ```
> from urllib.parse import urlparse
> 
> domain = urlparse(RSS_FEED_URLS["home-page"]).netloc
> ```
> 
> in the view.

Finally the pipfile made it:smiley:, also with the domain included

Reply:
Thanks! Apart from the superfluous comment, it looks good!